src/components/Auth/InputField/InputField.tsx

```diff
@@ -0,0 +1,87 @@
+// "use client";
+import { ChangeEvent, useState } from "react";
+import { EyeOffIcon, EyeOnIcon } from "@/src/asset";
```

Comment:
asset 폴더명은 복수형으로 표현해주는게 좋을 것 같아요~
```suggestion
import { EyeOffIcon, EyeOnIcon } from "@/src/assets";
```